go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/vitessio/arewefastyet/go/server
```

Comment:
Should be `module github.com/vitessio/arefastyet`.

Reply:
Your right, Will change that

Reply:
Sorry typo, `module github.com/vitessio/arewefastyet`